apps/charterafrica/src/components/EmbeddedDocumentViewer/EmbeddedDocumentViewer.js

```diff
@@ -0,0 +1,75 @@
+import { Section } from "@commons-ui/core";
+import { Box } from "@mui/material";
+import React from "react";
+
+import useDocument from "./useDocument";
+
+import { secondary, neutral } from "@/charterafrica/colors";
+import RichText from "@/charterafrica/components/RichText";
+
+const EmbeddedDocumentViewer = React.forwardRef(function EmbeddedDocumentViewer(
+  props,
+  ref
+) {
+  const { documentUrl, excerpt, showNotes, title, sx } = props;
```

Comment:
```suggestion
  const { excerpt, options, sx, title } = props;
```

Let `notes`, `search`, `zoom`, etc., be passed in as `options`.